bpmn-io/feel-editor
# feat: add basic highlighting and embedding support

![image](https://user-images.githubusercontent.com/21984219/172640225-69b2453c-3400-4cb5-a0de-4d7f1a405174.png)

To test the editor, run 
```
npm i
npm start
```

To test the embedded use-case in the properties panel, it is best to check out [the demo](https://marstamm.github.io/feel-editor-showcase/).
(Integration can change depending on the ongoing user tests. Direct `npx @bpmn-io/sr` unfortunately does not work without proper releases)

closes #1
